orchagent/bulker.h

```diff
@@ -104,6 +188,54 @@ namespace std
             return seed;
         }
     };
+
+    template <>
+    struct hash<sai_inbound_routing_entry_t>
+    {
+        size_t operator()(const sai_inbound_routing_entry_t& a) const noexcept
+        {
+            size_t seed = 0;
+            boost::hash_combine(seed, a.switch_id);
+            boost::hash_combine(seed, a.eni_id);
+            return seed;
```

Comment:
Given that the behavioral model uses ENI, VNI, and SIP as the lookup key for the inbound_routing table, do we need to add VNI and SIP here?

Reply:
I have made the change